### ISSUE TITLE
[C-877] Same member appearing twice in Organizations->Associated members

### ISSUE THREAD

**Issue by RaginiAngarka** [CrowdDotDev/crowd.dev#633]
### crowd.dev edition

Cloud (hosted)

### What happened?

Hello,
 
Steps to reproduce:-
1. Open workspace 'Hyperledger Workspace'
2. Click on 'Organizations' side nav
3. Go to 'Team Organizations' tab
4. Select the  'Linux Foundation Engineering' organization
5. Click on 'Associated Members' tab
6. Same member with same identity, same email address is appearing twice in the list. 

Actual Result:- Same member with same identity, same email address is appearing twice in the list. 

Expected Result:- There should not be two members with same identity, same email address etc and different activities.
Please refer attached screenshots.

<img width="1440" alt="Organizations" src="https://user-images.githubusercontent.com/87848481/223977404-e6b6d5af-82cd-494d-b9fb-700b0b84f82d.png">
<img width="1440" alt="Member - 0 Activities" src="https://user-images.githubusercontent.com/87848481/223977488-1ce9d332-acf6-4d7b-82ca-127078c56cce.png">
<img width="1440" alt="Member-707 Activities" src="https://user-images.githubusercontent.com/87848481/223977512-f215391f-6a00-46d0-89c7-a709068b13b2.png">

Please fix this. Thank you.

<sub>[C-877](https://linear.app/crowddotdev/issue/C-877/same-member-appearing-twice-in-organizations-associated-members)</sub>

**Comment by jonathimer** [comment]
@joanreyero added this to the upcoming cycle

**Comment by jonathimer** [comment]
but how can this happen? they should be automatically merged by us, no?

**Comment by joanreyero** [comment]
The data has a small latency when syncing. If the two members came very close together from integrations, it is possible we created both because, at that point, we were not aware it already existed. It's a rare problem, but it can happen. Once we have automatic merging, this will be sorted.

**Comment by joanreyero** [comment]
Bulk enrichment

**Comment by joanreyero** [comment]
Sorry, I'm just coming back to this now. I think @anil makes a good point.

**Comment by joanreyero** [comment]
cc @uros

**Comment by jonathimer** [comment]
We fixed the issue where member merge action might fail and leave the data in the database in a weird state like this (as Anil suggested). We also fixed a minor thing unrelated to this bug where this view still used the old slow endpoint for querying members.

If this was what happened with them, then yes, this has been fixed - now merge members will either fully succeed or fail with an error and all the changes will be rolled back as they were.
